faker/providers/internet/__init__.py

```diff
@@ -582,8 +582,18 @@ def ipv6(self, network: bool = False) -> str:
             address = str(IPv6Network(address, strict=False))
         return address
 
-    def mac_address(self) -> str:
-        mac = [self.generator.random.randint(0x00, 0xFF) for _ in range(0, 6)]
+    def mac_address(self, multicast: bool = False) -> str:
+        """
+        Returns a random MAC address.
+
+        :param multicast: Multicast address
+        :returns: MAC Address
+        """
+        mac = [self.generator.random.randint(0x00, 0xFF) for _ in range(0, 5)]
+        if multicast is True:
```

Comment:
Which advantages does this have over `if multicast:`?

Reply:
None, I just tried to follow the existing code style, example of the `ipv4` function:
```python
if private is True:
    return self.ipv4_private(address_class=address_class, network=network)
elif private is False:
    return self.ipv4_public(address_class=address_class, network=network)
```